src/main/java/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,34 @@
+package service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import service.DataConverter;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String REGEX = ",";
+    private static final int TYPE = 0;
+    private static final int FRUIT = 1;
+    private static final int QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> listOfFruitTransactions = new ArrayList<>();
+        for (int i = 1; i < lines.size(); i++) {
```

Comment:
Skipping the first line is a magic number usage. It's better to explain why you skip the first line (e.g., it might be headers). Consider defining a constant for the header line index or document this behavior.